### PR TITLE
Improved explanation of 'prices' in get_clean_factor_and_forward_returns

### PR DESCRIPTION
**Pull request by luca-s**
Related to issue #165

### CODE REVIEW

**Comment by luca-s** [comment]
@twiecki I'll let you improve the english ;)

File: alphalens/utils.py [hunk 1]
```diff
@@ -256,6 +256,15 @@ def get_clean_factor_and_forward_returns(factor,
         analysis time period plus an additional buffer window
         that is greater than the maximum number of expected periods
         in the forward returns calculations.
+        'Prices' must contains at least an entry for each date/asset
```
Comment (twiecki) [inline alphalens/utils.py]: contain

File: alphalens/utils.py [hunk 2]
```diff
@@ -256,6 +256,15 @@ def get_clean_factor_and_forward_returns(factor,
...
+        combination in 'factor'. This entry must be the asset price
+        at the time the asset factor value is computed and it will be
+        considered the buy price for that asset at that date.
+        'Prices' must also contains entries for dates following each
```
Comment (twiecki) [inline alphalens/utils.py]: contain

**Comment by twiecki** [comment]
Small typo but reads well otherwise!

**Comment by luca-s** [comment]
thanks, fixed.

**Pull request merged**